pkg/agent/agent.go

```diff
@@ -113,6 +116,16 @@ type Agent struct {
 
 	// Extra headers to inject into RPC requests to ambassador cloud.
 	rpcExtraHeaders []string
+
+	// diagnostics reporting
+	diagnosticsReportingStopped bool // Director stopped diagnostics reporting
+	//minDiagnosticsReportPeriod  time.Duration // How frequently do we collect diagnostics
+
+	// The state of reporting
+
+	//diagnosticsReportToSend   *agent.Diagnostics // Diagnostics report that's ready to send
```

Comment:
`diagnosticsReportToSend *agent.Diagnostics` meant to be left commented out and `// Diagnostics report that's ready to send` belongs on the following line?

Reply:
it will be removed. i was trying to follow how the snapshots were done, but decided to divert and not add the diagnostics report to the struct, instead pass it explicitly to the reporting function